### ISSUE TITLE
how to adjust learning rates or something to be better when using a pretrained model to train my dataset?please give me some advice

### ISSUE THREAD

**Issue by J-LINC** [eriklindernoren/PyTorch-YOLOv3#820]
I'm trying to train my own data set using pretrained weights, BDD100K. Can you give me some advice? (All parameters unchanged, e.g. learning rate and step of learning rate decay) Below is the result of my 12 epoch training. I think map is a little too low, but the values of each loss function shown in the diagram are very small. I am worried that if I continue without any hyper parameter changes, the model will be overfitting. I currently set the number of training rounds to 250 epoch, which is evaluated every 10 rounds, and every 50 epochs saves the model.The following image shows the losses I have trained for 10 epochs and the evaluation results on the validation set
![2023-03-22 23-13-28 的屏幕截图](https://user-images.githubusercontent.com/89594867/226950926-9b8825f5-a3fc-415f-adc2-57d0a3816b26.png)

![2023-03-22 23-16-25 的屏幕截图](https://user-images.githubusercontent.com/89594867/226951606-170e2a59-231c-48e1-bd17-50da79648d26.png)

**Comment by Flova** [comment]
What pretrained weights did you use? I would not worry about overfitting that much on this dataset scale. I am preparing a PR with better hyperparameters in the near future, but for now I would just recommend longer training and maybe a slightly higher lr.

**Comment by J-LINC** [comment]
> What pretrained weights did you use? I would not worry about overfitting that much on this dataset scale. I am preparing a PR with better hyperparameters in the near future, but for now I would just recommend longer training and maybe a slightly higher lr.

I did not use the pre-training model because I think BDD100k is big enough. I forgot whether the pre-training model was used when I trained COCO in the original paper. Now I am wondering whether to change the learning rate

**Comment by J-LINC** [comment]
> What pretrained weights did you use? I would not worry about overfitting that much on this dataset scale. I am preparing a PR with better hyperparameters in the near future, but for now I would just recommend longer training and maybe a slightly higher lr.

you mean learning rate > 0.001 will be better? i think the initial learning rate in the yolov3-custom.cfg is too low.

**Comment by J-LINC** [comment]
> What pretrained weights did you use? I would not worry about overfitting that much on this dataset scale. I am preparing a PR with better hyperparameters in the near future, but for now I would just recommend longer training and maybe a slightly higher lr.

Before your reply, I have modified what is not to use the pre-training model. As for the two messages I just replied to you, I plan to look at the training results tomorrow and reply to you

**Comment by J-LINC** [comment]
@Flova hi ,i think most issues about map low is because they just use one optimiser ,adam, i checked your code long time ago and i found sgd optimiser emmm,it 's better than adam in training ,at least in some aspects
